plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -59,9 +59,23 @@ export class GithubPublisher implements PublisherBase {
   }: PublisherOptions): Promise<PublisherResult> {
     const { owner, name } = parseGitUrl(values.storePath);
 
+    const { token } = await this.config.credentialsProvider.getCredentials({
+      url: values.storePath,
+    });
+
+    if (!token) {
```

Comment:
I'd personally throw an error here if there is no token returned.

`No token could be acquired for URL: ${values.storePath}` or something.

Reply:
@ebarriosjr maybe you missed this small change?

Don't return anything, just throw the error message and remove the log line in place of this error message instead.

Reply:
Done. :)